pkg/suggestion/suggest.go

```diff
@@ -0,0 +1,137 @@
+// Copyright © 2020 The Tekton Authors.
```

Comment:
Can you please add unit tests for `suggest.go`?

Reply:
thanks...added :slightly_smiling_face: